### ISSUE TITLE
No Activity found to handle Intent on some devices

### ISSUE THREAD

**Issue by csbenz** [lapism/SearchBar-SearchView#253]
Getting the following error on some devices, such as  Xiaomi, some android emulator,... 

`No Activity found to handle Intent { act=android.speech.action.RECOGNIZE_SPEECH (has extras) }`

When clicking on the voice button.

Here's the full error stack trace:

```
Fatal Exception: android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.speech.action.RECOGNIZE_SPEECH (has extras) }
       at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1545)
       at android.app.Instrumentation.execStartActivity(Instrumentation.java:1416)
       at android.app.Activity.startActivityForResult(Activity.java:3351)
       at android.support.v4.app.BaseFragmentActivityJB.startActivityForResult(BaseFragmentActivityJB.java:50)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:79)
       at android.app.Activity.startActivityForResult(Activity.java:3312)
       at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:859)
       at com.lapism.searchview.SearchView.onVoiceClicked(SearchView.java:1058)
       at com.lapism.searchview.SearchView.onClick(SearchView.java:1160)
       at android.view.View.performClick(View.java:4084)
       at android.view.View$PerformClick.run(View.java:16966)
       at android.os.Handler.handleCallback(Handler.java:615)
       at android.os.Handler.dispatchMessage(Handler.java:92)
       at android.os.Looper.loop(Looper.java:137)
       at android.app.ActivityThread.main(ActivityThread.java:4745)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:511)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by lapism** [comment]
Thanks I will try to fix it in next version 5.0.

**Comment by Chucky109** [comment]
Hi lapism,

I just faced the same problem. It happens when google voice search isn't installed or deactivated on the device. The mic icon is not shown at first but if you type in some text and delete it (backspace) then it appears and you can click on it => Crash.

Hope this might help you to fix it. If you already fixed then just ignore me :D Thanks for your lib!
